pkgs/tools/admin/awscli2/default.nix

```diff
@@ -94,7 +79,7 @@ with py.pkgs; buildPythonApplication rec {
     # https://github.com/NixOS/nixpkgs/issues/16144#issuecomment-225422439
     export HOME=$TMP
 
-    AWS_TEST_COMMAND=$out/bin/aws python scripts/ci/run-tests
+    # AWS_TEST_COMMAND=$out/bin/aws python scripts/ci/run-tests
```

Comment:
```suggestion
```

Reply:
So as you suggested I removed the whole line and thus the tests!

Reply:
These tests have been causing periodic issues on darwin also. https://github.com/NixOS/nixpkgs/issues/136214

Reply:
:+1: